app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -398,6 +398,16 @@ def zendesk_tags_video_url
       "https://forever1.zendesk.com/hc/en-us/articles/223595308-Using-Albums-and-Tags-Video-"
     end
 
+    def absolute_url(url_str)
```

Comment:
I think we should move this function above the urls because it is hidden so far down ... I think it would do better around line 31